.golangci.yml

```diff
@@ -48,8 +48,6 @@ issues:
     - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv). is not checked
     # The following are being worked on to remove their exclusion. This list should be reduced or go away all together over time.
     # If it is decided they will not be addressed they should be moved above this comment.
-    - (comment on exported (method|function|type|const)|should have( a package)? comment|comment should be of the form)
-    - package comment should be of the form "Package (.+) ..."
```

Comment:
Let's keep this exclusion? Seems a lot of comment are in the form of `package X implements X`, which isn't really super useful?

Reply:
I don't have strong opinions here.
Ideally we should get few lines explaining what each package is, but the linter just ensures a minimal comment exist, not that an actual explanation exists.
However, if we consider this as a starting point I'm ok with enabling this linter/this PR

Reply:
My worry is that we add these descriptions to satisfy the linter, and provide no information to our users.

Reply:
We could keep the regex for just packages, but I would actually like to keep this.

I agree most of the docstrings are not very helpful right now. But at least they have a basis to be improved upon. There are a couple cases where it actually did add some helpful information and caught where there were good godocs, but not formatted correctly to show up in IDE help because of where they were placed. And it would help make sure any new packages added at least force the authors to think about whether there is anything helpful to add to the godocs.

So bottom line, the added value is questionable, but it at least has potential for helping without much of a downside. Does that make sense to you @vincepri?